app/javascript/packs/data/socialMediaLinks.js

```diff
@@ -1,11 +1,11 @@
 export const FACEBOOK_LINK =
-  "https://www.facebook.com/sharer/sharer.php?u=https%3A//feedadoc.com/";
+  "https://www.facebook.com/sharer/sharer.php?u=https%3A//http://hospitalhero.care/";
```

Comment:
looks like these links are `https://http://hospitalhero.care` if I'm reading that correctly

Reply:
Thanks!